library/dialog.js

```diff
@@ -660,7 +680,7 @@ function dlgopen(url, winname, width, height, forceNewWindow, title, opts) {
             }
         } else {
             //if no buttons defined by user, add a standard close button.
-            oFoot.append('<button class="closeBtn btn btn-default" data-dismiss=modal type=button><?php echo xlt("Close") ?></button>');
+            oFoot.append('<button class="closeBtn btn btn-default" data-dismiss=modal type=button>Close</button>');
```

Comment:
Is this Close string something that is used a lot; or just there in case. Just trying to figure out if we need to figure out a way to incorporate translation of it (where it looks like you have figured out that calling php in a js script doesn't work).

Reply:
never gets seen and was there in case I need to have for emodal replacement. I think the translations go back to when I was going to load as php script and kept loading the same as previous(js) and never removed. Besides, i've only seen one error that would need to require xlation in whole time i've used since creating. Yes, i'm that good:)